crocket/src/main/java/com/crocket/model/surface/Surface.java

```diff
@@ -0,0 +1,34 @@
+package com.crocket.model.surface;
+
+public abstract class Surface {
+    private int size = 100;
+    private int xPosition;
```

Comment:
I would suggest renaming these to x and y to follow the naming scheme of an earlier pull request that changed this

Reply:
Checked the other PR, we still work with xPosition, yPosition, changed in Hitbox